### ISSUE TITLE
Concerns and clarifications with Accessible Authentication 3.3.7

### ISSUE THREAD

**Issue by simonrjones** [w3c/wcag#1470]
Hi, I agree with the concerns in issue #1364 - though I applaud efforts to make the login process easier to use (as an able user I find many login forms infuriating!).

I have a lot of experience building auth systems and I agree making complex changes would be very challenging for smaller website owners. If this is required would agree making this Level A feels unreasonable & would likely cause some anguish in the community.

I have to admit I’m not completely clear what auth methods do pass this criteria so if it’s OK I’ll run through my understanding to check some things: 

It sounds like a login form that supports password managers is OK. I would add that copy & paste should not be disabled since that’s how many users use these tools. 

It sounds like username is preferred over email. Many modern systems do not store a username so this is would be a near impossible requirement for many users to meet. 

I would also add login forms should not change input. I have experienced many login (or registration) forms that do things like strip spaces or cut character length. This then makes the login more difficult since there is confusion on what the password is. 

It sounds like time-based 2FA is not allowed under this criteria? Which I think covers all 2FA codes I’m aware of. If so, that feels like it would break a large swathe of the web. Also, 2FA is a really really good idea for web security so I would hesitate saying it’s a bad thing (if you are). Is this an example of accessibility vs security?

I appreciate other options exist but OAuth is complex to implement. WebAuthn looks great, but I have no experience with it (I understand Apple login uses this). Implementing something like OAuth or WebAuthn would be a pretty complex job - though a good idea for larger systems.

**Comment by alastc** [comment]
Hi @simonrjones,

Based on previous feedback there is a new [draft of the understanding document](https://w3c.github.io/wcag/understanding/accessible-authentication.html) which I think addresses your concerns, could you have a look please?

-Alastair

**Comment by alastc** [comment]
Draft response for discussion with the group (very similar to #1364):
<hr>
From the initial comment there appear to be some misunderstandings of what will pass/fail the criterion, PR #1499 has been added to make updates. The initial one is an example of resetting a password via email. The other examples already show how 2FA can be accomplished without relying on transcription.

With the comment above and the fact this SC will go through a security review at W3C, we think that addresses the general point about the restrictiveness of the SC.

**Comment by alastc** [comment]
Hi @simonrjones,

The group approved the response above. Please do re-open if you don't consider the issue addressed.
